src/server/game/Entities/Conversation/Conversation.cpp

```diff
@@ -138,6 +140,16 @@ bool Conversation::Create(ObjectGuid::LowType lowGuid, uint32 conversationEntry,
     for (ConversationLineTemplate const* line : conversationTemplate->Lines)
         SetDynamicStructuredValue(CONVERSATION_DYNAMIC_FIELD_LINES, linesIndex++, line);
 
+    sScriptMgr->OnConversationCreate(this, creator);
+
+    // return if there is no actor added (actors start at idx = 0), missing actors cause client crash
+    if (!GetDynamicStructuredValue<ConversationDynamicFieldActor>(CONVERSATION_DYNAMIC_FIELD_ACTORS, 0))
+        return false;
+
+    // option to prevent AddToMap by external script, e.g. ConversationScript or InstanceScript
+    if (!_isValid)
```

Comment:
like i already said to you, don't use such boolean that the user have to set
the crash does not happen cause of missing actor on index 0, it happens because of the line does not have an actor
you need to verifiy that your conversation have actors for all lines, if that fails, add a log message

Reply:
Since the data is taken by sniff and it is unkown how many actors are there until the developer adds actors, i donno how to Check it if you dont know it.

Reply:
you know it, actors depend on lines

Reply:
Ah yeah ok. Gonna add a Check for it.